client/index.html

```diff
@@ -60,6 +60,10 @@ <h2 class="title">The Lounge is loading…</h2>
 					</div>
 				</div>
 				<div id="sign-in" class="window">
+					<div class="header">
+						<span class="title">The Lounge</span>
+						<span class="topic">Web IRC Client</span>
```

Comment:
Would you mind just displaying "The Lounge"? I'm not too fond of that subtitle and don't think we need any. After all, if you're logging into the app, you've been past the presentation website...